client/src/actions/types.js

```diff
@@ -20,6 +20,7 @@ export const CREATE_BULK_PUBLICATIONS = 'CREATE_BULK_PUBLICATIONS';
 export const FETCH_TEAM_INFO = 'FETCH_TEAM_INFO';
 export const LINK_TEAM_TWITTER = 'LINK_TEAM_TWITTER';
 export const UNLINK_TEAM_TWITTER = 'UNLINK_TEAM_TWITTER';
+export const UPDATE_TEAM = 'UPDATE_TEAN';
```

Comment:
Spelling mistake here

Reply:
Fixed. Took me 5 mins to find the `N`. XD